packages/jaspr/analysis_options.yaml

```diff
@@ -3,5 +3,5 @@ include: package:lints/recommended.yaml
 linter:
   rules:
     sort_pub_dependencies: true
-    prefer_relative_imports: true
```

Comment:
I don't like relative imports I prefer write only absolute imports path.
Because sometimes you can somewhere see something like: `import '../../../../something.dart'`
For example after some refactoring or anything similar..
So better is write absolute paths everywhere I think for better readability and consistency with other imports which are not relative because are from different package for example..

Reply:
Absolute paths can get long too. This just comes down to personal preference. 
I would keep it as it is right now.

Reply:
I am not talking about long of imports path but I talk about readability.